src/components/Currency/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const Currency = props => {
+  const { value, showType, showSymbol, type, currencyClass } = props;
```

Comment:
Change showType to withCurrencyLabel because showType sounds like a method name and it is not clear what type is, for the same reason chnge showSymbol  to withSymbol.

Reply:
Change type to currency, for the reasons listed above.

Reply:
Please remove currencyClass property, the rest of the properties can be spread.